src/components/leaderboards/List.vue

```diff
@@ -1,52 +1,54 @@
 <script setup lang="ts">
 import { useAsyncState } from '@vueuse/core'
-import { computed, ref } from 'vue'
+import { ref } from 'vue'
 import { Leaderboards } from '../../lib/api/Leaderboards'
+import { StatusFilter } from '../../lib/api/data-contracts'
 
-enum IncludeDeleted {
-	NotDeleted,
-	Deleted,
-	All
-}
-
-const includeDeleted = ref(IncludeDeleted.NotDeleted)
-const search = ref<string>('')
+const limit = 256;
+const status = ref<StatusFilter>('Published')
+const query = ref<string>('')
```

Comment:
the `<string>` here is unnecessary as giving the `ref` function an initial value of an empty string TS wise we know the value is `string`.